README.md

```diff
@@ -16,7 +16,7 @@ Then install this package. Maven-style:
 
 Sbt-style:
 
-    dependencies += "org.overviewproject" %% "pdfocr" % "0.0.2"
+    libraryDependencies += "org.overviewproject" % "pdfocr_2.11" % "0.0.3"
```

Comment:
Hi there,

Thanks! You've made three changes, and two of them are great :).

The change from `dependencies` to `libraryDependencies` is correct, certainly.

The change from `%% "pdfocr"` to `% "pdfocr_2.11"` is [not](http://www.scala-sbt.org/0.13/docs/Library-Dependencies.html).

The change from "0.0.2" to "0.0.3" is great, too :).

Could you please correct the pull request?

Enjoy life,
Adam